todo-mvc/package.json

```diff
@@ -23,6 +27,7 @@
     "grunt-text-replace": "^0.4.0",
     "grunt-ts": "^5.5.1",
     "grunt-tslint": "^3.1.0",
+    "grunt-typings": "^0.1.5",
```

Comment:
That's the `grunt-typings` grunt tasks rather than typings... Either way it is not required, it is a dependency of `grunt-dojo2`